relationships/synthesis/EXT-SERVICE-to-REDIS.yml

```diff
@@ -1,5 +1,5 @@
 relationships:
-  - name: extServiceCallsInfraElastiCacheRedisCluster
+  - name: extServiceCallsRedis
```

Comment:
Shouldn't we limit this to just ElastiCache Redis as we do for APM?
If that's the case, we need to introduce a condition checking that `net.peer.name` has the format of `*.cache.amazonaws.com`

Reply:
I will be re-checking the regex and will be updating the file. ASAP

Reply:
I'm not following how does this message relate with my comment... Were you referring to [this other comment](https://github.com/newrelic/entity-definitions/pull/1653/files#r1633032030)?

This is the synthesis rule for `EXT-SERVICE` (OTel service) and we should apply the same conditions than in the synthesis rule for APM. For APM we're filtering out (using the regex) those redis endpoints that don't follow this format: `*.cache.amazonaws.com`. We should do the same here adding an extra condition.